api/v1/qdrantcluster_types.go

```diff
@@ -162,6 +171,9 @@ type QdrantClusterSpec struct {
 	// to inspect the attached storage.
 	// +optional
 	StartupDelaySeconds *int `json:"startupDelaySeconds,omitempty"`
+	// RebalanceStrategy specifies the strategy to use for automaticially rebalancing shards the cluster.
+	// +optional
```

Comment:
what about specifying that this is only taken into account if the cluster-manager (see property at line 114) is switch on (explicit or via the operator config)

Reply:
Done